Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.5
+FROM python:3.6
```

Comment:
Why the version change?  As far as I know, this is still using 3.5 in production.

Reply:
because there was a confict here, in Travis file the Python mentioned was 3.6, I've removed this change, and changed the Python to 3.5 in Travis file instead.